content/docs/reference/cookies.mdx

```diff
@@ -180,38 +186,17 @@ This setting cannot be set to `false` if [**Cookie SameSite**](#cookie-samesite)
 <Tabs>
 <TabItem value="Core" label="Core">
 
-| **Config file keys** | **Environment variables** | **Type**  | **Default** |
-| :------------------- | :------------------------ | :-------- | :---------- |
-| `cookie_secure`      | `COOKIE_SECURE`           | `boolean` | `true`      |
-
-#### Examples {#cookie-secure-examples}
-
-```yaml
-cookie_secure: false
-```
-
-```bash
-COOKIE_SECURE=false
-```
+As of **v0.25.0**, we've deprecated the **Cookie Secure** setting in Pomerium Core and will remove it in **v0.26.0**. Now, this setting will be enabled by default.
 
 </TabItem>
 <TabItem value="Enterprise" label="Enterprise">
 
-As of v0.25, we've removed the **Cookie Secure** setting from the Enterprise Console (in the Console, this setting was called **HTTPS only**).
+As of **v0.25.0**, we've removed the **Cookie Secure** setting from the Enterprise Console (in the Console, this setting was called **HTTPS only**). Now, this setting will be enabled by default.
```

Comment:
nit: adding the ".0" here may be misleading; the UI was removed in Console v0.25.0 but the corresponding backend change is pending a v0.25.1 patch release.

In general, I think it might be better to reference just the minor release (v0.25) unless we're referring specifically to a patch release.